pkgs/misc/vscode-extensions/default.nix

```diff
@@ -28,2174 +28,7 @@ let
-          name = "viml";
-          publisher = "xadillax";
-          version = "1.0.1";
-          sha256 = "sha256-mzf2PBSbvmgPjchyKmTaf3nASUi5/S9Djpoeh0y8gH0=";
-        };
-        meta = with lib; {
-          license = licenses.mit;
-        };
-      };
-
-      xaver.clang-format = buildVscodeMarketplaceExtension {
-        mktplcRef = {
-          name = "clang-format";
-          publisher = "xaver";
-          version = "1.9.0";
-          sha256 = "abd0ef9176eff864f278c548c944032b8f4d8ec97d9ac6e7383d60c92e258c2f";
-        };
-        meta = with lib; {
-          license = licenses.mit;
-          maintainers = [ maintainers.zeratax ];
-        };
-      };
-
-      xyz.local-history = buildVscodeMarketplaceExtension {
-        mktplcRef = {
-          name = "local-history";
-          publisher = "xyz";
-          version = "1.8.1";
-          sha256 = "1mfmnbdv76nvwg4xs3rgsqbxk8hw9zr1b61har9c3pbk9r4cay7v";
-        };
-        meta = {
-          license = lib.licenses.mit;
-        };
-      };
-
-      yzhang.markdown-all-in-one = buildVscodeMarketplaceExtension {
-        mktplcRef = {
-          name = "markdown-all-in-one";
-          publisher = "yzhang";
-          version = "3.4.0";
-          sha256 = "0ihfrsg2sc8d441a2lkc453zbw1jcpadmmkbkaf42x9b9cipd5qb";
-        };
-        meta = {
-          license = lib.licenses.mit;
-        };
-      };
-
-      zhuangtongfa.material-theme = buildVscodeMarketplaceExtension {
-        mktplcRef = {
-          name = "material-theme";
-          publisher = "zhuangtongfa";
-          version = "3.13.17";
-          sha256 = "100riqnvc2j315i1lvnwxmgga17s369xxvds5skgnk2yi2xnm2g9";
-        };
-        meta = {
-          license = lib.licenses.mit;
-        };
-      };
-
-      llvm-org.lldb-vscode = llvmPackages_8.lldb;
-
-      WakaTime.vscode-wakatime = callPackage ./wakatime { };
-
-      wholroyd.jinja = buildVscodeMarketplaceExtension {
-        mktplcRef = {
-          name = "jinja";
-          publisher = "wholroyd";
-          version = "0.0.8";
-          sha256 = "1ln9gly5bb7nvbziilnay4q448h9npdh7sd9xy277122h0qawkci";
-        };
-        meta = {
-          license = lib.licenses.mit;
-        };
-      };
-
-      zxh404.vscode-proto3 = buildVscodeMarketplaceExtension {
-        mktplcRef = {
-          name = "vscode-proto3";
-          publisher = "zxh404";
-          version = "0.5.4";
-          sha256 = "08dfl5h1k6s542qw5qx2czm1wb37ck9w2vpjz44kp2az352nmksb";
-        };
-        nativeBuildInputs = [ jq moreutils ];
-        postInstall = ''
-          cd "$out/$installPrefix"
-          jq '.contributes.configuration.properties.protoc.properties.path.default = "${protobuf}/bin/protoc"' package.json | sponge package.json
-        '';
-        meta = {
-          license = lib.licenses.mit;
-        };
-      };
-    };
+    (callPackage ./_maintainers/generated.nix { inherit buildVscodeMarketplaceExtension; });
 
   aliases = self: super: {
```

Comment:
do you mind adding an `overrides.nix`, should be applied similar to aliases overlay

```
  overrides = callPackage ./overrides.nix { };
  ...
  overlays = [
    overrides
  ] ++ lib.optionals (config.allowAliases or true) [
    aliases
  ];
```

Reply:
yes, I was planning on something similar, will do